CMakeLists.txt

```diff
@@ -36,16 +36,6 @@ if (MSVC)
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -wd4125")
 endif()
 
-# Define preprocessor for accelerators.
-set(NNPA_ENABLED 0)
-if (ACCELERATORS_TO_BUILD)
-  foreach(t ${ACCELERATORS_TO_BUILD})
-    if (${t} STREQUAL "NNPA")
-      set(NNPA_ENABLED 1)
-    endif()
-  endforeach(t)
-endif(ACCELERATORS_TO_BUILD)
```

Comment:
NNPA_ENABLED is used to trigger lit tests for NNPA in `test/mlir/accelerators/nnpa` folder. It's my mistake not updating the comment that is confusing.

Reply:
Yes I just saw that. I'm trying to see if there is a better way to do this.